code/modules/vehicles/armored/__armored.dm

```diff
@@ -70,8 +69,12 @@
 	var/zoom_mode = FALSE
 	/// damage done by rams
 	var/ram_damage = 20
+	/**  List for storing all item typepaths that we may "easy load" into the tank by attacking its entrance
+	 This will be turned into a typeCache on  initialize */
```

Comment:
/**
 * List for storing all item typepaths that we may "easy load" into the tank by attacking its entrance
 * This will be turned into a typeCache on  initialize
*/

Reply:
Github fucks with the indentation so bad here